### DISCUSSION TITLE
Broken API client layout

### DISCUSSION THREAD

**Discussion by Amorim33** [scalar/scalar#1686] [Q&A] [answered]
API client layout is breaking when the URL is too long

![image](https://github.com/scalar/scalar/assets/42624869/993acb56-ed34-44c5-a301-7e0c16036db0)

**Comment by hanspagel** [comment]
Great bug report! Let’s fix this! I’ve created an issue: https://github.com/scalar/scalar/issues/1687

**Reply by cameronrohani** [reply, marked as answer]
Made a PR here, let's get it in ASAP :D https://github.com/scalar/scalar/pull/1690
